httpobs/docs/scoring.md

```diff
@@ -137,12 +140,17 @@ x-xss-protection-enabled | `X-XSS-Protection` header set to `1` | 0
 x-xss-protection-disabled | `X-XSS-Protection` header set to `0` (disabled) | -10
 x-xss-protection-not-implemented | `X-XSS-Protection` header not implemented | -10
 x-xss-protection-header-invalid | `X-XSS-Protection` header cannot be recognized | -10
-
 <br>
 
 [Referrer-Policy](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Referrer-Policy) | Description | Modifier
 --- | --- | :---:
-referrer-policy-header-enabled | `Referrer-Policy` header enabled | 5
+referrer-policy-header-no-referrer | `Referrer-Policy` header set to `no-referrer` | 5
+referrer-policy-header-no-referrer-when-downgrade | `Referrer-Policy` header set to `no-referrer-when-downgrade` | 5
```

Comment:
The names should just be referrer-policy-foo and referrer-policy-bar.  The only one that should have header is referrer-policy-header-invalid, like the others.